showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -96,6 +96,7 @@ class ShowkaseProcessor @JvmOverloads constructor(
 
 
     private fun processPreviewAnnotation(roundEnvironment: XRoundEnv): Set<ShowkaseMetadata.Component> {
```

Comment:
I wonder if we should enable doing this repeatable annotation support even for `ShowkaseComposable` 🤔 What do you think? I'm on the fence about it.

Reply:
@oas004 From my perspective, the goal for `ShowkaseComposable` was to offer everything `Preview` offers + more functionality that we'd need on top of that. When I use this lens, I feel like allowing ShowkaseComposable to behave in the same manner might be valuable

Reply:
Yeah when seeing it from that perspective I agree. Also I think the consumer might expect the same behaviour as the `Preview` annotation 🤔 I tried to add support for it in b109238 😄 It only works with KSP as the stacked preview as this has the same issue with KAPT.